build-test.sh

```diff
@@ -317,7 +317,10 @@ build_Tests()
     fi
 
     if [ $__SkipNative != 1 ]; then
+        export CROSSCOMPILEREAL=${CROSSCOMPILE}
```

Comment:
What are these being used for?

Reply:
`CROSSCOMPILEREAL` is used to just temporarily save value of `CROSSCOMPILE`. Logic here is now the same as in build.sh, where `CROSSCOMPILE` is set to 0 before build_native

Reply:
I am confused why we would unset crosscompile here for the native build. It seems like that is where it is actually needed.

Reply:
You were right, this was an incorrect change. Correct fix was to properly setup portable build parameter. I've updated PR.